src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ConditionalOrder.java

```diff
@@ -70,4 +71,18 @@
     )
     private List<ListValue<DivorceDocument>> documentsUploaded;
 
+    @CCD(
+        label = "Solicitor’s name"
+    )
+    private String sotName;
+
+    @CCD(
+        label = "Solicitor’s firm"
+    )
+    private String sotFirm;
```

Comment:
should be solicitorFirm rather than statement of truth firm ? same goes for other fields